### PR TITLE
Add Home 6

### PR DESCRIPTION
**Pull request by p1eDT**

### CODE REVIEW

File: GitHome6/Patient.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHome6
+{
+    public class Patient : Person
+    {
+
```
Comment (AliaksandrNestsiarenka) [inline GitHome6/Patient.cs:11]: Please, remove redundant blank spaces between class name and method name

File: GitHome6/Patient.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        public IlnessType IlnessType { get; set; }
+
+
+        public Patient(string name, IlnessType ilnessType) : base(name)
+        {
+            IlnessType = ilnessType;
+        }
+
+        public void Show()
```
Comment (AliaksandrNestsiarenka) [inline GitHome6/Patient.cs:20]: Do we need this mehtod?

Comment (AliaksandrNestsiarenka) [reply GitHome6/Patient.cs:20]: We already have mehtod ShowInfo()

**Review by AliaksandrNestsiarenka — APPROVED** [review]

File: GitHome6/Patient.cs (hunk 1 above)
Comment (p1eDT) [reply GitHome6/Patient.cs:11]: done

File: GitHome6/Patient.cs (hunk 2 above)
Comment (p1eDT) [reply GitHome6/Patient.cs]: done. Removed methods

**Pull request merged**